app/models/spree/variant.rb

```diff
@@ -59,6 +59,8 @@ class Variant < ActiveRecord::Base
       %w(weight volume).include?(variant.product.andand.variant_unit)
     }
 
+    validates :unit_value, numericality: { greater_than: 0 }
```

Comment:
The power of Rails I suppose ;)

Reply:
Indeed! :point_right: https://guides.rubyonrails.org/v5.2/active_record_validations.html

Reply:
we should enforce that at DB level as well. There's no guarantee otherwise.